apps/Open-Typer/install-32

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+version=5.2.0
+
+#Install Depends
+install_packages libfuse2 || exit 1
```

Comment:
don't directly do this.
call `enable_module fuse`. refer to other pi-apps scripts

also I personally prefer one `install` script if the entire script is duplicated between 32/64bit. refer to other pi-app scripts for how to use `case` with the arch variable

Reply:
Alright, I'll do that right now.

Reply:
Pushing now, I hope the permission scopes are correct.